application/app/models/download_file.rb

```diff
@@ -4,15 +4,12 @@ class DownloadFile < ApplicationDiskRecord
   include ActiveModel::Model
   include LoggingCommon
 
-  ATTRIBUTES = %w[id collection_id type filename status size metadata].freeze
+  ATTRIBUTES = %w[id collection_id type filename status size creation_date start_date end_date metadata].freeze
   TYPES = %w[dataverse].freeze
-  STATUS = %w[ready downloading success error].freeze
 
   attr_accessor *ATTRIBUTES
 
-  validates_presence_of *ATTRIBUTES
```

Comment:
why we are no longer validating the presence of the attributes when we save a file?

Reply:
Fixed